### ISSUE TITLE
Unable to save part way through

### ISSUE THREAD

**Issue by rc16** [rw251/code-list-creator#5]
It would be useful to be able to save the algorithm part way through, and come back to it another time.

**Comment by rw251** [comment]
Yes. I think it should auto-save on every change - and make it clear to the user that this has happened - and instructions on how to resume when they return to the list.
